core/src/main/java/com/sequenceiq/cloudbreak/init/clustertemplate/DefaultClusterTemplateCache.java

```diff
@@ -122,12 +132,69 @@ private void loadByClasspathPath(Collection<String> names) {
                 });
     }
 
-    private void convertToClusterTemplate(String templateAsString) throws IOException {
+    private void convertToClusterTemplate(String clusterTemplateName, String templateAsString,
```

Comment:
this is not really a converter, as it has no return value, could you rename this?